src/main/java/uk/ac/ebi/embl/flatfile/writer/FlatFileWriter.java

```diff
@@ -57,6 +57,8 @@ public abstract class FlatFileWriter {
 	 */
 	private Integer optimalLineLength;
 
+	private Integer customMaximumLineLength;
```

Comment:
I would just call this:
```
maximumLineLength
```
like the
```
optimalLineLength
```

One of these is not more custom than the other.